bitcoin_client/ledger_bitcoin/wallet.py

```diff
@@ -58,34 +59,52 @@ def n_keys(self) -> int:
         return len(self.keys_info)
 
     def serialize(self) -> bytes:
-        keys_info_hashes = map(lambda k: element_hash(k.encode()), self.keys_info)
+        keys_info_hashes = map(
+            lambda k: element_hash(k.encode()), self.keys_info)
 
-        descriptor_template_sha256 = sha256(self.descriptor_template.encode()).digest()
+        descriptor_template_sha256 = sha256(
+            self.descriptor_template.encode()).digest()
 
         return b"".join([
             super().serialize(),
             write_varint(len(self.descriptor_template.encode())),
-            self.descriptor_template.encode() if self.version == WalletType.WALLET_POLICY_V1 else descriptor_template_sha256,
+            self.descriptor_template.encode(
+            ) if self.version == WalletType.WALLET_POLICY_V1 else descriptor_template_sha256,
             write_varint(len(self.keys_info)),
             MerkleTree(keys_info_hashes).root
         ])
 
-    def get_descriptor(self, change: bool) -> str:
+    def get_descriptor(self, change: Union[bool, None]) -> str:
```

Comment:
Yeah, it's semantically equivalent; I preferred this version because the behavior with `None` is completely distinct from the `True`/`False` ones, as opposed to a situation where a parameter can be omitted (e.g. because there's a default value).

Reply:
Yeah fair point vs. a `Optional[bool] = None` kwarg.